alfred/files/alfred.init

```diff
@@ -103,41 +94,38 @@ alfred_start()
 	return 0
 }
 
-start()
-{
+start_service() {
 	config_load "alfred"
 	config_foreach alfred_start alfred
-
-	if [ "$enable" = "0" ]; then
```

Comment:
I do not think this can be removed. It is set in the original file.

Reply:
Just had a closer look:
"config_foreach alfred_start alfred" (L109) runs alfred_start for each alfred uci config entry.
After alfred_start went through successfully, enable is set to 1 (L84).
With the if clause, we make sure that alfred is not started if:
- no alfred uci config is present
- a config which is present does not go through setup successfully.

So, we "need" the "if [ "$enable" = "0" ]; then exit 0 fi" except alfred is supposed to start in any case (i.e. without config).